streams/examples/src/main/java/org/apache/kafka/streams/examples/wordcount/WordCountTransformerDemo.java

```diff
@@ -80,17 +83,17 @@ public void init(final ProcessorContext context) {
                                 final KeyValue<String, Integer> entry = iter.next();
 
                                 System.out.println("[" + entry.key + ", " + entry.value + "]");
-
-                                context.forward(entry.key, entry.value.toString());
+                                final Record<String, String> record = new Record<>(entry.key, entry.value.toString(), timestamp);
```

Comment:
nit: we could just inline the `new Record(...)` in the `context.forward()`

Reply:
@bbejeck Thank you for your advice. I've already done it.